documentation/github_feature_request_template.md

```diff
@@ -0,0 +1,52 @@
+# Feature Request Template
+
+## **Title**
+Provide a clear and descriptive title for the feature request.
+
+## **Description**
+A detailed description of the feature request, including the problem it aims to solve and the benefits it will bring.
+
+## **Motivation**
+Explain why this feature is necessary and how it will improve the project.
+
+## **Proposed Solution**
+Describe the proposed solution in detail. Include any relevant code snippets, diagrams, or mockups.
+
+## **Alternatives Considered**
+List any alternative solutions or approaches that were considered and explain why they were not chosen.
+
+## **Additional Context**
+Add any other context or information that might be useful, such as related issues, links to discussions, or references to external resources.
+
+## **Checklist**
+- [ ] I have searched for existing feature requests that might be related to this one.
+- [ ] I have provided a detailed description of the feature request.
+- [ ] I have included all necessary information about the motivation and proposed solution.
+- [ ] I have attached relevant diagrams, mockups, or code snippets.
+
+## **Labels**
+Assign appropriate labels to categorize the feature request (e.g., enhancement, new feature).
+
+## **Assignees**
+Assign the feature request to relevant team members.
+
+## **Milestones**
+- [ ] Task 1
+- [ ] Task 2
+- [ ] Task 3
```

Comment:
Yes, but there's a difference between asking people to add a todo in the feature request (which we should certainly encourage) and connecting a feature request (issue) to a milestone, right (which we should also encourage). But this looks like a bit of both, no? Or am I missing something?

Reply:
My bad. I actually meant actionable tasks.